Signal/src/ViewControllers/InviteFlow.swift

```diff
@@ -236,28 +236,26 @@ class InviteFlow: NSObject, MFMessageComposeViewControllerDelegate, MFMailCompos
         mailComposeViewController.setMessageBody(body, isHTML: false)
 
         self.presentingViewController.dismiss(animated: true) {
-            self.presentingViewController.navigationController?.present(mailComposeViewController, animated: true) {
-                UIUtil.applySignalAppearence()
```

Comment:
We don't need to apply `applySignalAppearence`.  `ContactsPicker` uses Signal appearance.